### PR TITLE
update perf setup script for latest helm changes

### PR DESCRIPTION
**Pull request by ayj**

### CODE REVIEW

**Comment by ayj** [comment]
cc @mandarjog

File: perf/istio/setup.sh [hunk 1]
```diff
@@ -7,63 +7,78 @@ function download() {
...
   local release="${2:?"release"}"
-  
+
   local outfile="$(download ${DIRNAME} ${release})"
 
   if [[ ! -d "${DIRNAME}/${release}" ]];then
       tar -xzf "${outfile}" -C "${DIRNAME}"
       mv "${DIRNAME}/istio-${release}/install/kubernetes/helm" "${DIRNAME}/${release}"
       rm -Rf "${DIRNAME}/istio-${release}"
+      helm dep update --skip-refresh "${DIRNAME}/${release}/istio"
```
Comment (mandarjog) [inline perf/istio/setup.sh]: There is a helm update later as well
1. Probably remove the other one
2. what is the purpose of --skip-refresh ?

Comment (ayj) [reply perf/istio/setup.sh]: I get the following error with it.

```bash
$ helm dep update tmp/release-1.1-20181108-09-15/istio
Hang tight while we grab the latest from your chart repositories...
...Unable to get an update from the "local" chart repository (http://127.0.0.1:8879/charts):
        Get http://127.0.0.1:8879/charts/index.yaml: dial tcp 127.0.0.1:8879: connect: connection refused
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 14 charts
Deleting outdated charts

```

File: perf/istio/setup.sh [hunk 2]
```diff
@@ -7,63 +7,76 @@ function download() {
...
   local release="${2:?"release"}"
-  
+
   local outfile="$(download ${DIRNAME} ${release})"
 
   if [[ ! -d "${DIRNAME}/${release}" ]];then
       tar -xzf "${outfile}" -C "${DIRNAME}"
       mv "${DIRNAME}/istio-${release}/install/kubernetes/helm" "${DIRNAME}/${release}"
       rm -Rf "${DIRNAME}/istio-${release}"
+      helm dep update "${DIRNAME}/${release}/istio"
```
Comment (mandarjog) [inline perf/istio/setup.sh]: I still want this to work with 1.0.3, which does not have sub charts.

Comment (ayj) [reply perf/istio/setup.sh]: I made this step optional. ptal

**Pull request merged**